onionshare_gui/receive_mode/uploads.py

```diff
@@ -299,22 +280,116 @@ def reset(self):
         """
         Reset the uploads back to zero
         """
-        self.common.log('Uploads', 'reset')
         for upload in self.uploads.values():
-            upload.close()
```

Comment:
@micahflee the removal of `upload.close()` here I think introduced the regression of seeing 'old' uploads per my review. This was necessary in the old code to actually pop an upload out of being displayed permanently on 'clear history'

Reply:
Confirmed in my tests that re-instating `upload.close()` here fixes the widgets from re-appearing/overlapping each other.

Upload widget is different to Download widget because it's not just a progress bar, it's a collection of child widgets. That's why it's not enough to just close its child progress bar.